README.md

```diff
@@ -1,43 +1,23 @@
-# Adobe Experience Platform - Media extension for Android
+# Adobe Experience Platform Edge Media Mobile Extension - Android
```

Comment:
for consistency with the other SDKs we can use `Adobe Experience Platform Media for Edge Network Extension`

Reply:
I checked the edgeIdentity and edgeConsent and followed this